src/odoc/indexing.ml

```diff
@@ -23,21 +23,29 @@ let parse_input_files input =
     (Ok []) input
   >>= fun files -> Ok (List.concat files)
 
-let compile_to_json ~output ~occurrences hierarchies =
+let compile_to_json ~output ~occurrences ~wrap ~simplified hierarchies =
   let output_channel =
     Fs.Directory.mkdir_p (Fs.File.dirname output);
     open_out_bin (Fs.File.to_string output)
   in
   let output = Format.formatter_of_out_channel output_channel in
+  if wrap then Format.fprintf output "let documents = ";
+  Format.fprintf output "[";
```

Comment:
I'm not sure to understand this new `[`. It does not seem to be closed? And, it's independent from the new options: is it fixing a bug?

Reply:
Ok, my comment is already outdated :)

Reply:
Nope, dunno how that got there. It's in the commit on the branch that I thought I was previously testing - but it can't possibly work! A mystery!